lib/surveyor/nps/calculator.rb

```diff
@@ -92,4 +94,4 @@ def as_hash
       end
     end
   end
-end
+end
```

Comment:
@Huwj-Ones for next time, check your editor settings: this line indicates that your file is being saved without a newline at the end. (Every line in a text file should be terminated with a `\n`)